tracer/test/test-applications/integrations/dependency-libs/Samples.DatabaseHelper/RelationalDatabaseTestHarness.cs

```diff
@@ -170,26 +170,29 @@ private static async Task RunAsync(
                     Console.WriteLine("  Synchronous");
                     Console.WriteLine();
 
-                    command = commandFactory.GetCreateTableCommand(connection);
-                    commandExecutor.ExecuteNonQuery(command);
+                    using (var transaction = connection.BeginTransaction())
```

Comment:
We should probably test both with _and_ without a transaction. Can be a follow up PR though.

Reply:
Will do that in a follow up thanks!